internal/internal_workflow_client.go

```diff
@@ -63,6 +66,8 @@ const (
 	defaultGetHistoryTimeout = 65 * time.Second
 
 	getSystemInfoTimeout = 5 * time.Second
+
+	pollUpdateTimeout = 60 * time.Second
```

Comment:
For other polls, we make sure the server returns 10 seconds before this I believe. Can you confirm that will happen here?

Reply:
As it stands, that will not happen. Server prefers to return DeadlineExceeded here.

Reply:
> Server prefers to return DeadlineExceeded here.

I don't think that's true on normal long polls. I think it returns an empty success and I think it should here. Helps with error code tracking and such. I don't think it's an error case for your update to not be done yet.

Reply:
> I don't think that's true on normal long polls

This is correct. It was an intentional decision to use DEADLINE_EXCEEDED here and understood to be different than the queue polling calls.